lib/AST/Decl.cpp

```diff
@@ -3310,6 +3310,10 @@ bool ProtocolDecl::existentialTypeSupportedSlow(LazyResolver *resolver) {
   ProtocolDeclBits.ExistentialTypeSupportedValid = true;
   ProtocolDeclBits.ExistentialTypeSupported = true;
 
+  // Protocols imported from ObjC can always be existential.
+  if (isObjC() && hasClangNode())
```

Comment:
Protocols defined in Swift that are @objc can also be existential too.

Reply:
Ah, even better! So anything @ObjC